### DISCUSSION TITLE
Add tags to users

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco-CMS#15913] [Features and ideas]
As we get more projects on Umbraco Cloud and from time to time need to invite Umbraco Cloud support members to projects which are created in backoffice, but sometimes customers wonder who these people are.

We can disable user, but not delete.

I wonder if users could have a tags property similar to orders in Umbraco Commerce, then we could perhaps hook into a user creating notification and set a tag like "Umbraco HQ" on users with email `@umbraco.com` or `@umbraco.dk`

**Comment by bjarnef** [comment]
Perhaps the ability to filter users on tags as well as in Umbraco Commerce with orders. For example maybe a "Marketing" tag, but not necessary a "Marketing" group.

**Comment by markadrake** [comment]
I see a benefit to incorporating user types (or compositions we can assign) to add details like this.

**Reply by bjarnef** [reply]
I guess it could make sense in the long run, but probably requires users are based on `TEntity` like content, media and member.

If based on user types and possible composition I would require some properties to be locked as member types.
